### ISSUE TITLE
padding in canvas and  download image resolution

### ISSUE THREAD

**Issue by justraman** [kozakdenys/qr-code-styling#34]
Thanks for the great library.

I have a question.

![qr](https://user-images.githubusercontent.com/25192346/104130044-a09f6d00-5394-11eb-9655-02e6536ba384.png)

as you can see in the image I can't able to give a margin in the downloaded .png.  QR is visible in the corner of the image too.
how can I achieve this?

2nd  Question:  The image has the same resolution as the canvas has. Can we somehow download images with a better resolution without increasing the size of the canvas?

**Comment by justraman** [comment]
Anyways I solved the second issue with this.
````javascript
   qrStylingObj._canvas.getCanvas().style.height = '20rem'
  // add getCanvas getter to QRCodeStyling class so it can easily accessible. like qrStylingObj.getCanvas().style.height 
````

**Comment by glocore** [comment]
The library doesn't currently have an option to add a margin to the QR code itself.

> 2nd Question: The image has the same resolution as the canvas has. Can we somehow download images with a better resolution without increasing the size of the canvas?

You can achieve this by creating a second QRCodeStyling object with a larger width and height:
```ts
new QRCodeStyling({
  /* Your config */
  width: 2000,
  height: 2000,
}).download();
```

**Comment by kozakdenys** [comment]
Hi @justraman
1. About margins. The lib doesn't support it, but it is a good idea for improvement. 
The only hack I see to add some margin is to count horizontal or vertical dots (in your case is 25) and define the height and width of the canvas that will have the remainder of the division of 25. For example for height and width 324 % 25 === 24. So the margin will be 12 (24 divided by 2). Let me know if you need a better explanation.

2. You can init QRCodeStyling with any width and height, (2000 for example). Then change the width and height of canvas via css. So the downloaded file will have 2000x2000.

**Comment by justraman** [comment]
@kozakdenys 
 ## Thanks for your explanation

> The only hack I see to add some margin is to count horizontal or vertical dots (in your case is 25) 

It is 25 in this case but the `data` can be anything that's why the numbers of dots will possibly change. 
also adding a small margin is not enough. 
#35  I have created a PR for this seems like it's working for me. can you check yourself?

**Comment by kozakdenys** [comment]
Thanks, @justraman
Published your feature in version 1.3.4
